test/services/dumpers/agent-nodejs/agent-nodejs.test.ts

```diff
@@ -0,0 +1,478 @@
+
+        const osStub = jest.spyOn(os, 'platform').mockReturnValue('linux');
+
+        await dump();
+
+        const generatedFile = fs.readFileSync(
+          `${appRoot}/${appConfig.applicationName}/${fileName}`,
+          'utf-8',
+        );
+
+        const expectedFile = fs.readFileSync(
+          `${__dirname}/expected/${name}/${fileName === '.env' ? 'env' : fileName}`,
+          'utf-8',
+        );
+
+        expect(generatedFile).toStrictEqual(expectedFile);
+        rimraf.sync(`${appRoot}/${appConfig.applicationName}`);
+        osStub.mockRestore();
+      });
+    });
+
+    describe('on a linux based OS', () => {
+      describe('when the database is on the local machine', () => {
+        const setupAndDump = async (dbConnectionUrl: string | null = null) => {
+          const osStub = jest.spyOn(os, 'platform').mockReturnValue('linux');
+          const postgresConfig = {
+            appConfig: {
+              applicationName: 'test-output/postgres',
+              appHostname: 'localhost',
+              appPort: 1654,
+              path: appRoot,
+            },
+            dbConfig: {
+              dbDialect: 'postgres',
+              dbConnectionUrl: 'postgres://localhost:54369',
+              ssl: false,
+              dbSchema: 'public',
+            },
+            forestAuthSecret: 'forestAuthSecret',
+            forestEnvSecret: 'forestEnvSecret',
+          };
+          const injectedContext = Context.execute(defaultPlan);
+          const dumper = new AgentNodeJs(injectedContext);
+          await dumper.dump(
+            {
+              appConfig: postgresConfig.appConfig,
+              dbConfig: {
+                ...postgresConfig.dbConfig,
+                dbConnectionUrl: dbConnectionUrl || postgresConfig.dbConfig.dbConnectionUrl,
+              },
+              forestAuthSecret: 'forestAuthSecret',
+              forestEnvSecret: 'forestEnvSecret',
+            },
+            {},
+          );
+
+          return { osStub, postgresConfig };
+        };
+
+        const cleanStub = osStub => {
+          osStub.mockRestore();
+        };
+
+        it('should not make use of `host.docker.internal`', async () => {
+          expect.assertions(2);
+
+          const { osStub, postgresConfig } = await setupAndDump();
+          const dockerComposeFile = fs.readFileSync(
+            `${appRoot}/${postgresConfig.appConfig.applicationName}/docker-compose.yml`,
+            'utf-8',
+          );
+          const dotEnvFile = fs.readFileSync(
+            `${appRoot}/${postgresConfig.appConfig.applicationName}/.env`,
+            'utf-8',
+          );
+
+          expect(dockerComposeFile).not.toContain('DOCKER_DATABASE_URL');
+          expect(dotEnvFile).not.toContain('host.docker.internal');
+
+          rimraf.sync(`${appRoot}/${postgresConfig.appConfig.applicationName}`);
+          cleanStub(osStub);
+        });
+
+        describe('when the database is local on the system', () => {
```

Comment:
This describe is useless, let's move the `it` in the upper describe

Reply:
👍